README.md

```diff
@@ -553,3 +554,20 @@ Sorting fields include a compound index on on the filed to sort and a unique fie
 Database mirrors/replicas are supported by the API if the `SQLA_FOLLOWERS` is set to one or more valid connection strings.  By default, setting this environment variable will shift all `read` operations to any mirrors/replicas that are available (and randomly choose one to target per request if there are more than one).
 
 You can optionally choose to restrict traffic that goes to the mirrors/replicas to be the asynchronous tasks only by setting the `SQLA_RESTRICT_FOLLOWER_TRAFFIC_TO_TASKS` environment variable to something that will evaluate to `True` in Python (simply using `True` as the value is fine).  If you do this, you can also restrict which tasks are supported on the mirrors/replicas.  Supported tasks are configured by adding their fully qualified names to the `app.config['SQLALCHEMY_FOLLOWER_TASKS']` list in order to whitelist them.  By default, only the `download` task is enabled.
+
+### Database migration
+`flyway` is the tool used for database migration.
+
+#### Installing `flyway`
+Download the version without JRE, e.g. `flyway-commandline-4.2.0.tar.gz` and expand the archive. Add `<target_directory>/flyway/flyway-4.2.0` to your `PATH`.
+
+#### How `flyway` works
+All database schema modification code is checked into version control in the directory `data/migrations` in the form of SQL files that follow a strict naming convention - `V<version_number>__<descriptive_name>.sql`. `flyway` also maintains a table in the target database called `schema_version` which tracks the migration versions that have already been applied.
+
+`flyway` supports the following commmands:
+- `info` compares the migration SQL files and the table `schema_version` and reports on migrations that have been applied and those that are pending.
+- `migrate` compares the migration SQL files and the table `schema_version` and runs those migrations that are pending.
+- `baseline` modifies the `schema_version` table to indicate that the database has already been migrated to a baseline version.
```

Comment:
It might be useful to link to the full documentation after this list, e.g.,:

> For more information and a complete list of commands, [please see the full documentation of the command line tool](https://flywaydb.org/documentation/commandline/)

It took me a few clicks around the site to find that page, it's not completely straightforward!